mobile_app_connector/models/partner_communication.py

```diff
@@ -82,6 +82,9 @@ def send(self):
                     "destination": job.mobile_notification_destination,
                     "topic": job.mobile_notification_topic,
                     "partner_ids": [(4, job.partner_id.id)],
+                    "res_model": self.env["ir_model"].search(
+                        "name", 'like', object._name).id,
+                    "res_id": object._uid
```

Comment:
I would rather link it directly to the communication instead of the related objects.
```suggestion
                    "res_model": self._name,
                    "res_id": job.id
```